packages/frontend-shared/src/components/Tooltip.vue

```diff
@@ -46,7 +46,7 @@ const props = withDefaults(defineProps<{
   instantMove?: boolean
   showGroup?: string
 }>(), {
-  color: 'dark',
+  color: undefined,
```

Comment:
What did the component do with the `color` prop originally.  It does not look like it was used.  I am guessing it just got passed on to the `Menu` or `Tooltip` components from `floating-vue`.  I looked at the documentation for that library but could not find any documentation for it.  Did you look?

Reply:
Couldn't find any docs for it and it wasn't being referenced so I hijacked it!